client/views/admin/emailInbox/EmailInboxForm.js

```diff
@@ -0,0 +1,339 @@
+	};
+};
+
+export function EmailInboxEditWithData({ id }) {
+	const t = useTranslation();
+	const { value: data, error, phase: state } = useEndpointData(`email-inbox?_id=${ id }`);
+
+	if ([state].includes(AsyncStatePhase.LOADING)) {
+		return <FormSkeleton/>;
+	}
+
+	if (error || !data) {
+		return <Box mbs='x16'>{t('EmailInbox_not_found')}</Box>;
+	}
+
+	return <EmailInboxForm id={id} data={data} />;
+}
+
+export default function EmailInboxForm({ id, data }) {
+	const t = useTranslation();
+	const dispatchToastMessage = useToastMessageDispatch();
+	const setModal = useSetModal();
+	const { values, handlers, hasUnsavedChanges } = useForm(getInitialValues(data));
+
+	const {
+		handleActive,
+		handleName,
+		handleEmail,
+		handleDescription,
+		handleSenderInfo,
+		handleDepartment,
+		// SMTP
+		handleSmtpServer,
+		handleSmtpPort,
+		handleSmtpUsername,
+		handleSmtpPassword,
+		handleSmtpSslTls,
+		// IMAP
+		handleImapServer,
+		handleImapPort,
+		handleImapUsername,
+		handleImapPassword,
+		handleImapSslTls,
+	} = handlers;
+	const {
+		active,
+		name,
+		email,
+		description,
+		senderInfo,
+		department,
+		// SMTP
+		smtpServer,
+		smtpPort,
+		smtpUsername,
+		smtpPassword,
+		smtpSslTls,
+		// IMAP
+		imapServer,
+		imapPort,
+		imapUsername,
+		imapPassword,
+		imapSslTls,
+	} = values;
+
+	const router = useRoute('admin-email-inboxes');
+
+	const close = useCallback(() => router.push({}), [router]);
+
+	const saveEmailInbox = useEndpointAction('POST', 'email-inbox');
+	const deleteAction = useEndpointAction('DELETE', `email-inbox?_id=${ id }`);
+
+	const handleRemoveClick = useMutableCallback(async () => {
+		const result = await deleteAction();
+		if (result.success === true) {
+			close();
+		}
+	});
+
+	const handleDelete = useMutableCallback((e) => {
+		e.stopPropagation();
+		const onDeleteManager = async () => {
+			try {
+				await handleRemoveClick();
+				dispatchToastMessage({ type: 'success', message: t('Removed') });
+			} catch (error) {
+				dispatchToastMessage({ type: 'error', message: error });
+			}
+			setModal();
+		};
+
+		setModal(<DeleteWarningModal
+			onDelete={onDeleteManager}
+			onCancel={() => setModal()}
+		>{t('You_will_not_be_able_to_recover_channel')}</DeleteWarningModal>);
+	});
+
+	const handleSave = useMutableCallback(async () => {
+		const smtp = { server: smtpServer, port: smtpPort, username: smtpUsername, password: smtpPassword, sslTls: smtpSslTls };
+		const imap = { server: imapServer, port: imapPort, username: imapUsername, password: imapPassword, sslTls: imapSslTls };
+		const payload = { active, name, email, description, senderInfo, department, smtp, imap };
+		if (id) {
+			payload._id = id;
+		}
+		try {
+			await saveEmailInbox(payload);
+			dispatchToastMessage({ type: 'success', message: t('Saved') });
+			close();
+		} catch (e) {
+			dispatchToastMessage({ type: 'error', message: e });
+		}
+	});
+
+	const canSave = hasUnsavedChanges && name && (email && isEmail(email))
```

Comment:
When the user types a wrong email format nothing happens to the component, we are not doing the same we do on other forms, such as departments:

![Screen Shot 2021-01-08 at 15 49 15](https://user-images.githubusercontent.com/59577424/104052933-34244280-51c9-11eb-8b05-78b2bffe47cd.png)

We must do the same for duplicated emails.